tests/system/python/scripts/package/remove

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+time sudo apt purge -y foglamp foglamp-gui
```

Comment:
`foglamp-gui` package removal? Are we installing foglamp-gui from our system tests?

Reply:
foglamp-gui is getting installed from setup script.